[ADA-1088] Add community outreach content

Rwolfe-Nava opened the pull request:
We want to add community outreach content as outlined in ticket: 1088.
-Adds a new page: /communityoutreach
-Adds a link to underneath the contact section in the header for the new communityoutreach page.

Tested on mobile and desktop layouts.

## Screenshots
<img width="986" alt="Screenshot 2024-12-18 at 3 16 19 PM" src="https://github.com/user-attachments/assets/e571e370-bcf1-42d4-ae4f-b943efd031b2" />
<img width="467" alt="Screenshot 2024-12-18 at 3 15 27 PM" src="https://github.com/user-attachments/assets/eb39275a-6654-496b-84d2-5fcd12c7a3a7" />

## Preview Desktop Gif
![communityoutreachdemo](https://github.com/user-attachments/assets/ecd94619-c2f5-41fc-8f64-5a748c3fb72a)

## Preview Mobile Gif
![communityoutreach_mobiledemo](https://github.com/user-attachments/assets/581b9c22-c17a-41f3-a552-5576e59c9cc5)

c2nelson commented:
@Rwolfe-Nava The link is throwing a 404 error.  Could it be because you've populated a redirect?  This is a new page so a redirect isn't necessary.  Just a thought.

Rwolfe-Nava commented:
I noticed that as well with federalist. The page does not throw a 404 and all of the links work fine locally so its a bit tricky to nail down. 

Thank you for the insight though, I will try removing the redirect. If there is anything else you can think it might be please let me know

vwilliamsn commented:
@Rwolfe-Nava - thanks for your work on this. Can you add a video of what you're seeing locally? I'm also getting the 404 page and the federalist link shows a redirect (when it's a new page). https://www.ada.gov/preview/usdoj-crt/beta-ada/rw/1088-outreach-content/_pages/redirects/communityoutreach/

Rwolfe-Nava commented:
@vwilliamsn It makes sense that it is redirecting to the 404 page if it can't find the new page for some reason. I have removed the redirect that @c2nelson was referring to and it did not fix the problem in federalist. 

I added two GIFs to the original post description showing my local environment. One for desktop mode and one for mobile mode in my browser.

vwilliamsn commented:
> @vwilliamsn It makes sense that it is redirecting to the 404 page if it can't find the new page for some reason. I have removed the redirect that @c2nelson was referring to and it did not fix the problem in federalist.
> 
> I added two GIFs to the original post description showing my local environment. One for desktop mode and one for mobile mode in my browser.

@Rwolfe-Nava - thank you!!!

Rwolfe-Nava commented:
@okkays When you have time to review this please look into the 404 issue in the federalist for this new page. I modeled this off of your PR for the 'help' page, which doesn't seem to get a 404 in the federalist preview. So I am not sure if the 404 in this case is catching a bug in my code or not. 

It seems to function locally, but I am not sure how else we can test to make sure this is good to go. Thank you!

okkays commented:
@Rwolfe-Nava there is an existing redirect in place for `/communityoutreach`. These redirects don't happen locally (there are a lot of them, so we turn them off locally to be able to develop faster) but they do happen on preview. That's why it's pushing you to `ada.gov` instead of remaining on the preview domain, as well.

Deleting this file (which is no longer necessary) should fix it: `_pages/redirects/communityoutreach.md`

okkays reviewed (changes requested):
(Remove `_pages/redirects/communityoutreach.md`)

vwilliamsn commented:
@Rwolfe-Nava - per the mock, please add ![image](https://github.com/user-attachments/assets/abf47a38-aa47-433e-859d-0fd4f4f3efa9) under the left side navigation.

vwilliamsn commented:
> @vwilliamsn - Good catch. It has now been added and can been seen in federalist preview

Thanks @Rwolfe-Nava! Your preview gifs (above) show the icons in the Presentation/Participation box & Information Sharing box, however in the federalist link the icons are missing. Any idea why?

Rwolfe-Nava commented:
@vwilliamsn Thanks, I see that issue too. It looks like for some reason the images either do not exist on the federalist build, or it cannot find them. (It's returning a 404 error when trying to load the icons). Looking into it now.

Rwolfe-Nava commented:
@vwilliamsn  Just pushed a commit that seems to have fixed the issue with icons not loading properly in federalist.

vwilliamsn commented:
Hey @kavithakrishnan4 - can you take a quick look at the pull request. Please confirm that the spacing in the boxes align with your vision and specs for the page. Thanks!

kavithakrishnan4 commented:
@vwilliamsn @Rwolfe-Nava Everything looks good to me.

vwilliamsn reviewed (approved):
LGTM! Great job, Richard!

okkays reviewed (approved):
lgtm!
